src/keylessBackup/index.ts

```diff
@@ -81,3 +82,35 @@ export async function getEncryptedMnemonic({
   const { encryptedMnemonic } = (await response.json()) as { encryptedMnemonic: string }
   return encryptedMnemonic
 }
+
+export async function deleteEncryptedMnemonic(wallet: ViemWallet) {
+  if (!wallet.account) {
+    // This should never happen
+    throw new Error('no account found in the wallet')
+  }
+  const siweClient = new SiweClient(
+    {
+      accountAddress: wallet.account.address,
```

Comment:
shouldn't this use the encryption address and not the wallet address? (same as what is done in GET). The cab backend has no knowledge of the wallet address

Reply:
I am unable to delete my cloud backup despite a successful restore and receive a 404 response.

Reply:
I'm also unable to delete, getting a 404 not found as well. @satish-ravi What's required to get the encryption address? Looking at the existing code, it appears that we construct the encryption address by getting both keyshares (torus and valora), constructing the encryption private key, and converting that into the public key (address). We'd also need to sign the SIWE message using the encryption private key, so I suppose getting the public key sets us up for that as well. 

I guess it seems to me like this may change the UX somewhat, since the user will actually need to retrieve both keyshares in order to delete, right?

Reply:
Actually, after looking at the ticket (and the code), it appears that the recommendation is to just store their encryption private key in the keychain and lock it behind PIN entry?

Reply:
yes, I believe that is the recommendation @jophish